wesnoth-sota.po

```diff
@@ -4688,9 +4636,8 @@ msgstr "Poloviční prázdnota"
 #. [value]: type=SotA Flying Corpse_Bat
 #. shown instead of the unit_type name in the recruitment dialog
 #: data/campaigns/Secrets_of_the_Ancients/utils/zombie-utils.cfg:24
-#, fuzzy
 msgid "recruit^Flying Corpse (Bat)"
-msgstr "Létající mrtvola"
+msgstr "recruit^Flying Corpse (Netopýr)"
```

Comment:
Před ^ je poznámka, po ^ bychom měli překládat. Mýlím se?
```suggestion
msgstr "Létající mrtvola (netopýr)"
```

Reply:
No nebyl jsem si jistý, tak jsem to nějak zkusil.